lib/routes/home.js

```diff
@@ -53,7 +52,7 @@ module.exports = async app => {
   }
 
   return fullname().then(name => {
-    const allo = (name && isString(name)) ? `'Allo ${name.split(' ')[0]}! ` : '\'Allo! ';
+    const allo = (name && _.isString(name)) ? `'Allo ${name.split(' ')[0]}! ` : '\'Allo! ';
```

Comment:
I will occasionally inline lodash if it's a one-off and the specific method was imported directly.